app/feature/feature-terminate-insurance/src/main/kotlin/com/hedvig/android/feature/terminateinsurance/data/GetContractsToTerminateUseCase.kt

```diff
@@ -0,0 +1,63 @@
+package com.hedvig.android.feature.terminateinsurance.data
+
+import arrow.core.Either
+import arrow.core.raise.either
+import com.apollographql.apollo3.ApolloClient
+import com.hedvig.android.apollo.safeExecute
+import com.hedvig.android.apollo.toEither
+import com.hedvig.android.core.common.ErrorMessage
+import com.hedvig.android.core.uidata.UiMoney
+import com.hedvig.android.data.contract.ContractGroup
+import com.hedvig.android.data.contract.toContractGroup
+import com.hedvig.android.logger.logcat
+import kotlinx.datetime.LocalDate
+import octopus.ContractsToTerminateQuery
+
+internal interface GetContractsToTerminateUseCase {
+  suspend fun invoke(): Either<ErrorMessage, List<InsuranceForCancellation>>
+}
+
+internal class GetContractsToTerminateUseCaseImpl(
```

Comment:
Do you think the name `GetTerminatableContractsUseCase` would fit this class better?
My thought process is that the current name tells us that we are getting the contracts to terminate. That may imply that there's a decision of which contracts are supposed to be terminated, and not necessarily the contracts that are just *eligible* to be terminated.
Something like `GetContractsEligibleForTerminationUseCase` could work too, albeit while being quite more verbose.
Wdyt?